lib/Slider/SliderAccessibilitySupport.js

```diff
@@ -0,0 +1,33 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name SliderAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: ['value']}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (was, is, prop) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			if (this.popup) {
+				this.setAttribute('aria-label', enabled ? this.value : null);
+			} else {
+				this.setAttribute('role', enabled ? 'slider' : null);
+				this.setAttribute('aria-valuenow', enabled ? this.value : null);
+			}
+			this.setAttribute('tabindex', enabled ? 0 : null);
```

Comment:
Why do we need to set tabindex here when it's also done in enyo/AccessibilitySupport?